conference.js

```diff
@@ -2336,6 +2338,102 @@ export default {
         }
     },
 
+    /**
+     * Set a device to be currently used, selected by the user.
+     *
+     * @param {MediaDeviceInfo} device - The device to save.
+     * @returns {boolean} - Returns true in order notifications to be dismissed.
+     * @private
+     */
+    _useDevice(device) {
```

Comment:
I think this function can be easily moved somewhere in the react folder, isn't it?

Reply:
Well, this code should move to react when we refactor all device handling and remove it from conference.js. I can move it now as it is, like a utility function somewhere, by adding a new param passing dispatch function, but is not worth it at the moment and will be used only by non react code ... so its better to be moved when we do the refactor.

Reply:
The thing is, if we are only relying on the moment when we are going to refactor conference.js and we continue to put new code there, it makes it harder and harder to refactor later. 

I would agree to leave it here if moving it would involve some significant effort, time and refactoring. But those are new functions and new code. You can just copy paste them into the react folder and everything will continue working as it is working now. 

I don't see a reason why they need to be methods of the conference object since we are not using `this`.

I would also agree with your argument about keeping the device handling on a single place but I don't think this is the current situation anyway. Actually the device handling is all over the place - in conference.js, mediaDeviceHelper.js (which should also be moved into the react folder very soon) and in react/features/devices. For example we already use `setAudioInputDevice` and other functions from the react folder in conference.js.

Reply:
So you want it those to be moved to react/features/devices? 

I don't care at all where I will move it, cause those are currently used only in conference.js only and they do not belong to react folder and once we refactor it, those functions will most probably be deleted.